src/tally.F90

```diff
@@ -441,6 +441,67 @@ subroutine score_general_ce(p, t, start_index, filter_index, i_nuclide, &
         end if
 
 
+      case (SCORE_PROMPT_NU_FISSION)
+        if (t % estimator == ESTIMATOR_ANALOG) then
+          if (survival_biasing .or. p % fission) then
+            if (t % find_filter(FILTER_ENERGYOUT) > 0) then
+              ! Normally, we only need to make contributions to one scoring
+              ! bin. However, in the case of fission, since multiple fission
+              ! neutrons were emitted with different energies, multiple
+              ! outgoing energy bins may have been scored to. The following
+              ! logic treats this special case and results to multiple bins
+              call score_fission_eout_ce(p, t, score_index, score_bin)
+              cycle SCORE_LOOP
+            end if
+          end if
+          if (survival_biasing) then
+            ! No fission events occur if survival biasing is on -- need to
+            ! calculate fraction of absorptions that would have resulted in
+            ! prompt-nu-fission
+            if (micro_xs(p % event_nuclide) % absorption > ZERO) then
+                score = p % absorb_wgt * micro_xs(p % event_nuclide) % fission &
+                     * nuclides(p % event_nuclide) % nu(E, EMISSION_PROMPT) &
+                     / micro_xs(p % event_nuclide) % absorption
+            else
+              score = ZERO
+            end if
+          else
+            ! Skip any non-fission events
+            if (.not. p % fission) cycle SCORE_LOOP
+            ! If there is no outgoing energy filter, than we only need to
+            ! score to one bin. For the score to be 'analog', we need to
+            ! score the number of particles that were banked in the fission
+            ! bank as prompt neutrons. Since this was weighted by 1/keff, we
+            ! multiply by keff to get the proper score.
+            score = keff * p % wgt_bank * (1.0 - sum(p % n_delayed_bank) &
+                 / real(p % n_bank))
```

Comment:
`real(p % n_bank, 8)` is safer. Just worried about losing precision here converting integers to reals